src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -101,8 +102,11 @@ const getCategories = (
     }>
   >((ret, feature) => {
     const parentCategory =
-      feature.properties?.[boundaryLayer.adminLevelNames[1]];
-    const label = feature.properties?.[boundaryLayer.adminLevelNames[0]];
+      feature.properties?.[boundaryLayer.adminLevelNames[0]];
+    const label =
+      feature.properties?.[
+        last(boundaryLayer.adminLevelNames) || boundaryLayer.adminLevelNames[1]
```

Comment:
can you explain what's happening here? when would `last` return a nullish value here but `boundaryLayer.adminLevelNames[1]` would not?

Reply:
I took a shortcut here to make TS happy haha. `last` returns `string | undefined` which I needed to convert to `string`. I'll make this cleaner...

Reply:
You could say you were just testing my reviewing skills ;)